Test/API/SampleAPITest.robot

```diff
@@ -0,0 +1,63 @@
+*** Settings ***
+Documentation  API Testing
+Library  RequestsLibrary
+Resource  SampleAPIResource.robot
+
+Task Setup  Create New Sessions
```

Comment:
Unable to execute the tests, find the details below

**To Get List Of Users :: To Get List Of All Users                      | FAIL |
Setup failed:
No keyword with name 'Create New Sessions' found. Did you mean:
    RequestsLibrary.Create Session**

Reply:
Will Resolve path issue

Reply:
Path issue was fixed. Now It will run from all end